ocs_ci/framework/conf/default_config.yaml

```diff
@@ -36,6 +36,9 @@ REPORTING:
     address: "ocs-ci@redhat.com"
   polarion:
     project_id: "OpenShiftContainerStorage"
+  ocs_version: '4.2'
```

Comment:
I would rather not specify such values in config files. This could backfire because:
- it has no actual meaning (OCS version is specified by various image versions), when this value is updated, it has no effect 
- it's creates information duplicity, which is prone to various human errors

Moreover when we have ocs version operator, it will have full control over this and the only canonical way to get version would be to ask it in the same way, as we can ask cluster version operator now.

Reply:
Maybe you can rename it to "test run prefix" or something like that, and drop it from the default config file.

Reply:
Fixed by 0b2ca9ff.